.travis.yml

```diff
@@ -6,13 +6,18 @@ cache:
   pip: true
   directories:
     - "$HOME/.npm"
+    - "$HOME/.cache"
 # cf. https://stackoverflow.com/a/52387639
 install:
 - travis_retry gem install s3_website -v 3.4.0
 - travis_retry pip install awscli --upgrade --user
 - travis_retry npm ci
+- cypress install
```

Comment:
I think you can/should use `travis_retry` here for consistency with the others.

Reply:
Fixed!